bedrock/base/templates/includes/protocol/footer/footer-newsletter.html

```diff
@@ -0,0 +1,25 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+<div class="moz24-newsletter-container">
+  <div class="moz24-newsletter-wrapper">
+    <div class="moz24-newsletter-info">
+      <img class="moz24-newsletter-image" src="{{ static('img/logos/mozilla/footer-refresh/symbol.svg') }}" alt="mozilla symbol" width="40" height="48">
+      <p>{{ ftl('footer-refresh-find-out-about') }}</p>
+    </div>
+
+    <div class="moz24-newsletter">
+      {{ email_newsletter_form(
+        newsletters='knowledge-is-power',
+        title=None
+        )}}
+      </div>
+    </div>
+</div>
+
+{% block js %}
+  {{ js_bundle('newsletter') }}
+{% endblock %}
```

Comment:
**Suggestion:** we try and keep JS scrips at the end of the page for performance, so this could be moved to the end of `footer-refresh.html`? (not sure you can add `defer` easily here). You could also rename the block to something more unique like `footer_js`.

At the end of `footer-refresh.html`:

```
{% block footer_js %}
  {% if show_newsletter %}
    {{ js_bundle('newsletter') }}
  {% endif %
{% endblock %}